content/blog/2015-06-03-service-discovery-introduction.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Service Discovery Introduction
+created_at: 2015-06-02
+kind: article
+author: Fabian Reinartz
+---
+
+This week we released Prometheus v0.14.0 — a version with many long-awaited additions
+and improvements.
+
+On the user side, new mechanisms for service discovery aim to make it easier to
+dynamically update the targets Prometheus scrapes. In addition to DNS-SRV records,
+[Consul](https://www.consul.io) is now supported out-of-the-box and a file-based interface
+allows you to connect your own discovery mechanism.
+Over time, we plan to extend Prometheus with native support for other common ways of doing
+service discovery.
+
+Aside from many smaller fixes and improvements, you can now also reload your configuration at 
+runtime by sending a `SIGHUP` to the Prometheus process. For a full list of changes, check the
+[changelog for this release](tba).
+
+In this blog post, we will take a closer look at the built-in service discovery mechanisms and provide
+small practical examples. As an additional resource
+[Prometheus's configuration documentation](/docs/operating/configuration),
+can be helpful.
+
+
+## Prometheus and targets
+
+For proper understanding of this blog post, we have to take a look at how Prometheus sees targets.
+To Prometheus each scrape target is nothing more but another set of labels – even its address, which
+is defined by the `__address__` label.
```

Comment:
This seems like too much technical/theoretical detail for the beginning of the post. I'd start with some usage examples of different SD mechanisms and then dive into the more theoretical and technical underpinnings...